### PR TITLE
Image above 10MB shows default image icon as preview

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

Any image above 10 MB shows default image icon as the preview instead.

## Test Plan

N/A

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: app/config/storage/logos.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 return [ // Based on this list @see http://stackoverflow.com/a/4212908/2299554
     'default' => __DIR__.'/logos/none.png',
+    'default_image' => __DIR__.'/logos/image.png',
```
Comment (TorstenDittmann) [inline app/config/storage/logos.php:5]: Can we stick a little to the Mime type naming and do something like `image/*`? 🤔

File: docs/references/storage/get-file-preview.md [hunk 2]
```diff
@@ -1 +1 @@
-Get a file preview image. Currently, this method supports preview for image files (jpg, png, and gif), other supported formats, like pdf, docs, slides, and spreadsheets, will return the file icon image. You can also pass query string arguments for cutting and resizing your preview image.
\ No newline at end of file
+Get a file preview image. Currently, this method supports preview for image files (jpg, png, and gif), other supported formats, like pdf, docs, slides, and spreadsheets, will return the file icon image. You can also pass query string arguments for cutting and resizing your preview image. We do not support preview for file size above 10MB.
```
Comment (eldadfux) [inline docs/references/storage/get-file-preview.md:1]:
```suggestion
Get a file preview image. Currently, this method supports preview for image files (jpg, png, and gif), other supported formats, like pdf, docs, slides, and spreadsheets, will return the file icon image. You can also pass query string arguments for cutting and resizing your preview image. Preview is supported only for image files smaller than 10MB.
```

**Review by eldadfux — CHANGES_REQUESTED** [review]
We should have consistent icons. Let's use @sarakaandorp new icons and deprecate file types that don't have a dedicated icon and make them use the default file icon.

**Review by TorstenDittmann — APPROVED** [review]

**Comment by lohanidamodar** [comment]
> We should have consistent icons. Let's use @sarakaandorp new icons and deprecate file types that don't have a dedicated icon and make them use the default file icon.

@eldadfux for image icon here I'm using Sara's new icons, and rest icons are already updated in another PR here
https://github.com/appwrite/appwrite/pull/2774

File: app/config/storage/logos.php (hunk 1 above)
Comment (lohanidamodar) [reply app/config/storage/logos.php:5]: Why, this is not the icon based on mime type, but rather a default icon for images larger than 10 mb, for image mimes we show preview?

**Comment by lohanidamodar** [comment]
Here's how it looks
image

**Comment by lohanidamodar** [comment]
Here's how it looks
image

**Review by eldadfux — APPROVED** [review]

**Pull request merged**